programmers/두-개-뽑아서-더하기.js

```diff
@@ -0,0 +1,16 @@
+function solution(numbers) {
+    
+    const len = numbers.length;
+    let result = [];
+    let add;
+    
+    for (let i = 0; i < len - 1; i++){
+        for (let j = i + 1; j < len; j++){
+            add = numbers[i] + numbers[j];
+            if (result.includes(add) === false)
```

Comment:
```suggestion
            if (!result.includes(add))
```
false나 true같은 boolean값과 비교할때는 
!를 하면 false와 같다고 비교
result.includes(add)같이 그냥 쓰게 되면 true랑 비교하게 됩니다.

Reply:
확실히 !를 사용하니까 코드가 더 간결해지는군요... 리뷰 덕분에 변수명 지정하는거랑 선언에 관한 내용, 제가 사용했던 함수식에서도 어떤 부분에서 발전해야 하는지 보였습니다. 오늘도 리뷰 감사합니다!

Reply:
아이고 감사합니다  화이팅!! 🤗